### PR TITLE
creating the bio at any cost

### PR DESCRIPTION
**Pull request by Rus604**
I have no idea why this became so difficult.  first i could not  " push" to git hub. so i dragged and dropped into the master..  only to realize that was the wrong order ( develop first then merge with master)  but what is done is done.  I think this should show up for your review.  

below is the error i received from my computer.

'''
Rich Russell@EVE-COMMAND ~/lab00-Rus604 (develop)
$ git push --set-upstream origin develop
Username for 'https://github.com': Rus604
Password for 'https://Rus604@github.com':
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/msu-csc131/lab00-Rus604/'
'''
I feel my bio is worth a review. and a smirk

### CODE REVIEW

**Comment by professordaehn** [comment]
Yeah, it's hard to say what may have gone wrong. From what I can tell it looked legit.

File: bio.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# PERSPECTIVE 
+
+![Image of How Many](https://github.com/msu-csc131/lab00-Rus604/blob/master/threeORfour.jpg)
+
+
+## Can you relate?
```
Comment (professordaehn) [inline bio.md]: My cousins used punch cards, so yes, I actually can relate!

File: bio.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# PERSPECTIVE 
+
+![Image of How Many](https://github.com/msu-csc131/lab00-Rus604/blob/master/threeORfour.jpg)
+
+
+## Can you relate?
+    1. In highschool I was programing in fortan on a punch card machine.
```
Comment (professordaehn) [inline bio.md]: For me it was two sizes; 5.25 and 3.5.. and then came the mighty ZipDrive!

File: bio.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# PERSPECTIVE 
+
+![Image of How Many](https://github.com/msu-csc131/lab00-Rus604/blob/master/threeORfour.jpg)
+
+
+## Can you relate?
+    1. In highschool I was programing in fortan on a punch card machine.
+    2. "Floppy disk" comes in THREE sizes.
```
Comment (professordaehn) [inline bio.md]: I had an IBM PS/2 Model 30

File: bio.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# PERSPECTIVE 
+
+![Image of How Many](https://github.com/msu-csc131/lab00-Rus604/blob/master/threeORfour.jpg)
+
+
+## Can you relate?
+    1. In highschool I was programing in fortan on a punch card machine.
+    2. "Floppy disk" comes in THREE sizes.  
+    3. In college I owned a KAYPRO PC.
```
Comment (professordaehn) [inline bio.md]: I could smoke a cigarette waiting for my C++ programs to compile when I was in graduate school...

**Comment by professordaehn** [comment]
Looks like you merged in the wrong direction! But you know that already...

**Comment by Rus604** [comment]
I am working on the pycharm and hope to have it finished tonight.. I'll try an get the assignment loaded in the correct direction.. 😊

________________________________

**Pull request merged**